tests/mvp-demo/mvp-demo.go

```diff
@@ -49,6 +50,8 @@ const (
 	managedNamespace    = "mvp-managed"
 
 	appName = "mvp-test-app"
+	BundleURL              = "quay.io/redhat-appstudio/example-tekton-bundle:integration-pipeline-pass"
```

Comment:
@dheerajodha @jinqi7 Would we want to use the new one Kruno put together for the eph namespace env? It simulates a real test? or do we plan on swapping it out later?

Reply:
@Danny, the new one for the eph namespace, including env & some credentials. This PR is to add a IntegrationTestScenario test case without env settings. They are different cases.

Reply:
Yes, I have another PR (https://github.com/redhat-appstudio/e2e-tests/pull/496) open which will use the new bundle by Kruno and update this test to use the ephemeral environment.